src/PAModel/PAConvert/Yaml/YamlLexer.cs

```diff
@@ -509,6 +515,25 @@ private YamlToken Unsupported(LineParser line, string message)
             return Error(line, message);
         }
 
+        private YamlToken UnsupportedSingleQuote(LineParser line, int lastSingleQuoteIndex, bool isComponent)
+        {
+            string[] lineSplit = line._line.ToString().Split(new string[] { "As" }, StringSplitOptions.None);
```

Comment:
Small thing, but I'd recommend doing a `ToLower()` here and comparing against "as" just in case the casing on this changes at some point.